assembl/static2/js/app/components/debate/common/postActions.jsx

```diff
@@ -147,4 +167,5 @@ class PostActions extends React.Component {
   }
 }
 
+// $FlowFixMe
```

Comment:
what is this?

Reply:
it is the ignore flag for flow. Apparently, flow types are not handled correctly in this case (it is related to the types def in react-apollo)